src/modules/billing/jobs/rebilling.js

```diff
@@ -49,7 +51,7 @@ const handler = async job => {
 
   // Process each invoice that needs rebilling
   for (const invoice of invoices) {
-    await invoiceService.rebillInvoice(batch, invoice);
+    await rebillingService.rebillInvoice(batch, invoice.id);
```

Comment:
Is the await here useful/necessary?

Reply:
It means that the tasks are processed in series (avoids overwhelming our DB/CM) and should avoid uncaught errors